template/page-studs/index/with-tw.tsx

```diff
@@ -7,6 +7,18 @@ type TechnologyCardProps = {
   documentation: string;
 };
 
+export type Variant<T extends string> = { [P in T]: string };
```

Comment:
Since this part of code is meant to demonstrate the usage of a design-pattern for re-usable and tokenized components, an explanation line (preferably pointing to links to somewhere in the README explaining why components have been separated like this) would be a great addition!